src/Tribe/RSVP.php

```diff
@@ -1420,10 +1424,54 @@ public function send_tickets_email_legacy( $order_id, $event_id = null ) {
 	 * Dispatches a confirmation email that acknowledges the user has RSVP'd
 	 * in cases where they have indicated that they will *not* be attending.
 	 *
+	 * @since TBD Adjusted the method to use the new Tickets Emails Handler.
+	 *
 	 * @param int $order_id The order ID.
 	 * @param int $event_id The event ID.
 	 */
 	public function send_non_attendance_confirmation( $order_id, $event_id ) {
+		if ( ! tec_tickets_emails_is_enabled() ) {
+			return $this->send_non_attendance_confirmation_legacy( $order_id, $event_id );
+		}
+
+		$attendees = $this->get_attendees_by_order_id( $order_id );
+
+		if ( empty( $attendees ) ) {
+			return;
+		}
+
+		// For now all ticket holders in an order share the same email.
+		$to = $attendees['0']['holder_email'];
+
+		if ( ! is_email( $to ) ) {
+			return;
+		}
+
+		$email_class = tribe( TEC\Tickets\Emails\Email\RSVP_Not_Going::class );
+
+		if ( ! $email_class->is_enabled() ) {
+			return false;
+		}
+
+		$subject     = $email_class->get_subject();
+		$content     = $email_class->get_content( [ 'tickets' => $attendees ] );
+		$headers     = $email_class->get_headers();
+		$attachments = $email_class->get_attachments();
+
+		$sent = tribe( TEC\Tickets\Emails\Email_Sender::class )->send( $to, $subject, $content, $headers, $attachments );
```

Comment:
Are we supposed to return the `$sent` variable?

Reply:
The[ legacy system it's not sending it](https://github.com/the-events-calendar/event-tickets/pull/2586/files#diff-356814ea7b647ee8d3827cc02717ad3870c4a021a1c59e2fe3d3618cd10e53c4R1568), but we could totally add it, in case it's useful.